data/discovery/register/citizens-advice-bureau.yaml

```diff
@@ -0,0 +1,13 @@
+fields:
+- citizens-advice-bureau
+- name
+- address
+- citizens-advice-bureau-membership-number
```

Comment:
If the membership number is used as the unique ID (`citizens-advice-bureau` field) then this can be removed.